### DISCUSSION TITLE
Cannot connect to PSQL database created by `npx mikro-orm schema:create -r`

### DISCUSSION THREAD

**Discussion by tukusejssirs** [mikro-orm/mikro-orm#2534] [Q&A] [answered]
1. I created a (test) database schema (structure) using `npx mikro-orm schema:create -r` both on my local machine and on a production server (both are running Linux).
2. Then I connected to PostgreSQL server via `psql` command as `postgres` user.
3. When I list all available databases (`\l`), the database is there both on local machine and the server.
4. But when I try to connect to it (`\c db_name`), I can only do that on local machine. On the server, it errors out with `FATAL: no such database: db_name`.

I even tried to create the database prior to creating the schema, but the error is the same. I also tried to grant all permissions on the database to `postgres`.

What could be the problem?

**Comment by B4nan** [comment, marked as answer]
I doubt this would be something connected to MikroORM. My guess is you have some permissions issue, but can't really help with that.

**Reply by tukusejssirs** [reply]
Strange thing is that Mikro-ORM (via Nest.js app) _can_ connect to the database. Is there a way to mimick how Mikro-ORM connects to the server (low level)?

**Reply by B4nan** [reply]
There is nothing special, if you want to rule MikroORM out, use knex directly, or even pg directly. But I really don't think the issue is there, it will be more about you trying to use different user or something like that.

**Reply by tukusejssirs** [reply]
You were right from the beginning. I forgot that we currently use `pgbouncer` on that server and I tried to connect to the test DB via `pgbouncer`, which was configured to allow connections only to some databases.

Sorry for the noise.
